lmctl/client/api/assemblies.py

```diff
@@ -81,10 +90,13 @@ def _intent_request_impl(self, intent_name: str, intent_obj: Union[Dict,
                                                                         HealAssemblyIntent, 
                                                                         ScaleAssemblyIntent,
                                                                         UpgradeAssemblyIntent]) -> str:
+        process_intents = ["retry", "rollback", "cancel"]
         endpoint = self.intent_endpoint(intent_name)
         if isinstance(intent_obj, Intent):
             intent_obj_data = intent_obj.to_dict()
         else:
             intent_obj_data = intent_obj
         request = TNCOClientRequest(method='POST', endpoint=endpoint).add_json_body(intent_obj_data)
+        if intent_name in process_intents:
+            return self._exec_request(request)
```

Comment:
Has it been confirmed that retry/rollback/cancel do not return a location header?

Reply:
Posted question on channel for Aubrey and Niall 
https://ibm-cloud.slack.com/archives/C044EG83NTB/p1674555284861619

Reply:
Niall Confirmed that response header will not have any response header.
https://ibm-cloud.slack.com/archives/C044EG83NTB/p1674556568982089?thread_ts=1674555284.861619&cid=C044EG83NTB